airflow/providers/postgres/hooks/postgres.py

```diff
@@ -211,7 +211,7 @@ def get_iam_token(self, conn: Connection) -> Tuple[str, str, int]:
             token = aws_hook.conn.generate_db_auth_token(conn.host, port, conn.login)
         return login, token, port
 
-    def get_table_primary_key(self, table: str, schema: Optional[str] = "public") -> List[str]:
+    def get_table_primary_key(self, table: str, schema: Optional[str] = "public") -> Optional[List[str]]:
         """
         Helper method that returns the table primary key
 
```

Comment:
Hi. @subkanthi
Could you rename `table` to `schema` on lines 220:221?

I've also checked the Postgres operator and found this mistake, but after that, I saw that such PR already exists.

Reply:
done.